server/handlers/component_handler.go

```diff
@@ -1204,3 +1204,64 @@ func (h *Handler) RegisterMeshmodelComponents(rw http.ResponseWriter, r *http.Re
 	}
 	go h.config.MeshModelSummaryChannel.Publish()
 }
+
+// swagger:route GET /api/meshmodels/registrants GetMeshmodelModels idGetMeshmodelModels
```

Comment:
Thanks for writing these swagger annotaions. This is most helpful.

Please generate the API docs too.